org.robocode.Robocode.yml

```diff
@@ -0,0 +1,91 @@
+app-id: org.robocode.Robocode
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: robocode
+rename-appdata-file: robocode.appdata.xml
+rename-desktop-file: robocode.desktop
+rename-icon: robocode
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --env=PATH=/usr/bin:/app/bin:/app/jdk/bin
+  - --env=JAVA_HOME=/app/jdk
+  - --env=JAVA=/app/jdk/bin/java
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+        ROBOCODE_VERSION: '1.9.3.7'
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/installjdk.sh
+
+  - name: robocode
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+    sources:
+      # Fetch project.
+      - type: file
+        only-arches:
+          - x86_64
```

Comment:
Limit the builders arch's using flathub.json file, see here https://github.com/flathub/flathub/wiki/App-Maintenance#limiting-the-set-of-architectures-to-build-on

Reply:
This was left as a mistake. I'll remove it since it'd be cool to see if it's usable in ARM.
(Thanks for spotting it!)